client/src/app/site/motions/components/motion-comments/motion-comments.component.html

```diff
@@ -0,0 +1,75 @@
+<ng-container *ngIf="vp.isMobile ; then mobileView; else desktopView"></ng-container>
+
+<ng-template #mobileView>
+    <mat-expansion-panel *ngFor="let section of sections">
+        <mat-expansion-panel-header>
+            <mat-panel-title>
+                <mat-icon>comment</mat-icon>
+                {{ section.getTitle() }}
+            </mat-panel-title>
+        </mat-expansion-panel-header>
+        <ng-container *ngIf="!isCommentEdited(section)">
+            <ng-container *ngIf="comments[section.id]">
+                {{ comments[section.id].comment }}
+            </ng-container>
+            <span *ngIf="!comments[section.id] || !comments[section.id].comment" translate>
+                No comment<!--TODO: make this nice gray and centered with the css classes from #3919-->
+            </span>
+        </ng-container>
+
+        <form [formGroup]="commentForms[section.id]" *ngIf="isCommentEdited(section)">
+            <mat-form-field>
+                <textarea formControlName="comment" matInput placeholder="{{'Comment' | translate}}"></textarea>
+            </mat-form-field>
+        </form>
+
+        <mat-action-row *ngIf="canEditSection(section)">
+            <button mat-icon-button *ngIf="!isCommentEdited(section)" (click)="editComment(section)">
+                <mat-icon>edit</mat-icon>
+            </button>
+            <button mat-icon-button *ngIf="isCommentEdited(section)" (click)="saveComment(section)">
+                <mat-icon>save</mat-icon>
+            </button>
+            <button mat-icon-button *ngIf="isCommentEdited(section)" (click)="cancelEditing(section)">
+                <mat-icon>close</mat-icon>
+            </button>
+        </mat-action-row>
+    </mat-expansion-panel>
+</ng-template>
+
+<ng-template #desktopView>
+        <mat-card class="meta-text-block" *ngFor="let section of sections">
```

Comment:
this pretty much duplicates the "personal notes" part of motion-details, where the mobile version of this _kind of_ duplicates an entry in the motion-category and motion-comment.

We could convert this into an own component to re-use a lot of styling and functionality. Might require multi-slot-transclusion.